src/Symfony/Bundle/FrameworkBundle/Templating/Helper/CodeHelper.php

```diff
@@ -38,8 +33,7 @@ class CodeHelper extends Helper
     public function __construct($fileLinkFormat, string $projectDir, string $charset)
     {
         $this->fileLinkFormat = $fileLinkFormat ?: ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
-        $this->projectDir = str_replace('\\', '/', $projectDir).'/';
-        $this->rootDir = $this->projectDir;
+        $this->rootDir = str_replace('\\', '/', $projectDir).'/';
```

Comment:
I would keep the new name, it's easier to follow along.

Reply:
but changin it breaks BC...

Reply:
Yes and no. That's internal/obsolete code (templating vs Twig).